tests/functional/titers/cram/titers-sub-with-tree.t

```diff
@@ -17,3 +17,11 @@ Test titer substitution model with alignment and tree inputs.
    --- 272 total measurements
   $ grep cTiterSub $TMP/titers-sub.json | wc -l
   \s*120 (re)
+
+Verify that the titer drops assigned per branch correspond to the expected values for this dataset.
+In this example, we know that the HA1 amino acid sequence for A/Fujian/445/2003 carries a S193N substitution and that the titer model assigns a weight of 0.6 to that substitution.
+The titer model assigns a higher weight of 1.22 to the opposite substitution N193S.
+When we search for that sequence's per-branch titer drop, we should get the smaller value below.
+
+  $ grep -A 3 '"A/Fujian/445/2003": {' $TMP/titers-sub.json | grep dTiterSub
+  \s*"dTiterSub": 0.60.* (re)
```

Comment:
The first decimal point in `0.60` should be escaped so that the regex doesn't match any char in that position.

Alternatively, a clearer way to access/match the value might be:

```suggestion
  $ python3 -c 'import json, sys; print(json.load(sys.stdin)["nodes"]["A/Fujian/445/2003"]["dTiterSub"])' < $TMP/titers-sub.json
  0.60* (glob)
```

Reply:
Thanks, @tsibley! I agree that the alternate test is much clearer (and safer to run).